### PR TITLE
Cleanup js-ipfs mentions

### PR DESCRIPTION
**Pull request by ElPaisano**
WIP

As part of https://github.com/ipfs/ipfs-docs/issues/1611, makes changes to ipfs-docs as described for each page in the **Actions** column of https://pl-strflt.notion.site/ipfs-docs-js-ipfs-cleanup-d46e22253bec4497865c38e7a9277eac

@SgtPooki @whizzzkid you should be able to contribute directly to this PR

### CODE REVIEW

**Comment by BigLep** [comment]
@ElPaisano : I'm converting to draft given its WIP.  I'm happy to take a look once it has all the changes you intend to make.

**Comment by BigLep** [comment]
Hi guys - this has been open for a few weeks.  I'd like to find a more incremental approach so we can start removing js-ipfs mentions as we go rather than waiting for it all to be done.

**Comment by SgtPooki** [comment]
@ElPaisano To address @BigLep's latest comment; How can we get these changes merged in faster?

File: docs/concepts/faq.md [hunk 1]
```diff
@@ -34,6 +35,20 @@ When you add a file to IPFS, IPFS splits it into smaller blocks. IPFS hashes eac
 
 There are a lot of ways you can contribute to IPFS, whether you're interested in helping with either of the core implementations, applications like IPFS Desktop, writing or editing documentation, doing UX, or whatever you enjoy working on. [Get all the details on where to get started here.](../community/contribute/ways-to-contribute.md)
 
+### What is an implementation
```
Comment (BigLep) [inline docs/concepts/faq.md:38]: We should really be driving people towards https://specs.ipfs.tech in this section and particularly https://specs.ipfs.tech/architecture/

File: docs/concepts/faq.md [hunk 2]
```diff
@@ -34,6 +35,20 @@ When you add a file to IPFS, IPFS splits it into smaller blocks. IPFS hashes eac
...
+
+An IPFS implementation is any software with basic functionality for interaction with other IPFS implementations. An implementation:
+
+- Supports addressability using CIDs.
+
+- Exposes operations like retrieval, provisioning and indexing on resources using CIDs. The operations that an implementation may support are open-ended, but this requirement should cover any interaction which the implementation exposes to other IPFS implementations.
+
+- Verifies that the CIDs it resolves match the resources they address, at least when it has access to the resources bytes. However, implementations may relax this requirement in controlled environments in which it is possible to ascertain that verification has happened elsewhere in a trusted part of the system.
+
+There are already many IPFS implementations, including [Kubo](../install/command-line.md), [Helia](../install/js-ipfs.md), and [more](../concepts/ipfs-implementations.md).
```
Comment (BigLep) [inline docs/concepts/faq.md:48]: I think better to link to the Kubo and Helia project pages.

Also is js-ipfs.md still a thing?

File: docs/concepts/nodes.md [hunk 3]
```diff
@@ -25,128 +26,66 @@ You're likely to see the term _node_ throughout the IPFS docs, issues, and relat
...
-- All default bootstrap nodes are part of the public DHT.
-- The list of bootstrap nodes a JS-IPFS node connects to is configurable in their config files.
+- All default bootstrap _nodes_ are part of the public DHT.
+- The list of bootstrap _nodes_ a Helia _node_ connects to is controlled by configuring [libp2p's peerDiscovery option](https://github.com/libp2p/js-libp2p/blob/master/doc/CONFIGURATION.md#peer-discovery).
 
 #### Limitations of a bootstrap node:
 
-- If an IPFS node only has one bootstrap node listed in that configuration and that bootstrap node goes offline, the IPFS node will lose access to the public DHT if it were to restart.
+- If an IPFS _node_ only has one bootstrap _node_ listed in that configuration and that bootstrap node goes offline, the IPFS node will lose access to the public DHT if it were to restart.
+  - Note that you should be able to configure your _peer_ store in your implementation to cache healthy connectable _peers_ so you can connect to them again after a restart, instead of bootstrap _nodes_. [Kubo recently added support for this](https://github.com/ipfs/kubo/pull/8856).
```
Comment (BigLep) [inline docs/concepts/nodes.md:66]: [Kubo has support this](https://github.com/ipfs/kubo/issues/3926)

"recent" is relative
I think better to link to the issue which has the context and links to the PR

**Review by BigLep — COMMENTED** [review]
I left some comments.  I couldn't do "suggestions" because this was already merged.  @ElPaisano : are you ok to take on the followup review incorporating this feedback?

**Comment by ElPaisano** [comment]
Thanks for reviewing @BigLep these comments are being addressed in https://github.com/ipfs/ipfs-docs/pull/1639

**Pull request merged**